R/mouse/BASE.R

```diff
@@ -0,0 +1,129 @@
+#' Souce code for the BASE algorithm. 
+#' This code is adapted from Varn et al., DOI: 10.1158/0008-5472.CAN-16-2490 
+#' 
+#' @param 
+#---------------------------------------------------------------------------------------------------
+#	data: numeric matrix; Patient gene expression data, with genes as rows and patients as columns
+#	reg: numeric matrix; Reference immune cell weights created using SuppSoftware1
+#	perm: numeric; Number of permutations to perform during the normalization step
+#	myoutf: character; Directory location to save cell lineage score output file to (example: "/home/immune/cell_scores.txt")
+#	median.norm: logical;  parameter stating whether to median center the expression values for each gene
+#---------------------------------------------------------------------------------------------------
+
+base <- function(data, reg, perm=100, median.norm=T)
```

Comment:
could you find a different name for that function? 
`base` conflicts with the `base::` package in R, and even if it might not cause problems, it will at least cause confusion.

Reply:
updated to base_algorithm